packages/chain-adapters/src/ethereum/EthereumChainAdapter.test.ts

```diff
@@ -420,6 +420,47 @@ describe('EthereumChainAdapter', () => {
     })
   })
 
+  describe('signMessage', () => {
+    it('should sign a properly formatted signMessageInput object', async () => {
+      const args = makeChainAdapterArgs()
+      const adapter = new ethereum.ChainAdapter(args)
+      const wallet = await getWallet()
+      wallet.ethSignMessage = async () => ({
+        address: '0x41e5560054824ea6b0732e656e3ad64e20e94e45',
+        signature:
+          '0xf77391e26e80101e0126c5da3225693b3c84c25f6669460603a8eec1c876748572bf180fbdae3f0f93bca331c4c5d0ad6abe8da025249de976b162d0d1b1c0181c'
+      })
+
+      const message = {
+        wallet,
+        messageToSign: {
+          message: 'Hello world 111',
+          addressNList: [2147483692, 2147483708, 2147483648, 0, 0]
+        }
+      } as unknown as SignMessageInput<ETHSignMessage>
+
+      await expect(adapter.signMessage(message)).resolves.toEqual(
+        '0xf77391e26e80101e0126c5da3225693b3c84c25f6669460603a8eec1c876748572bf180fbdae3f0f93bca331c4c5d0ad6abe8da025249de976b162d0d1b1c0181c'
+      )
+    })
+
+    it('should throw if wallet.ethSignMessage returns null', async () => {
+      const args = makeChainAdapterArgs()
+      const adapter = new ethereum.ChainAdapter(args)
+      const wallet = await getWallet()
+      wallet.ethSignMessage = async () => null
+      const message = {
+        wallet,
+        messageToSign: {
+          message: 'Hello world 111',
+          addressNList: [2147483692, 2147483708, 2147483648, 0, 0]
+        }
+      } as unknown as SignMessageInput<ETHSignMessage>
```

Comment:
The `as unknown` is not needed here 
```suggestion
      } as SignMessageInput<ETHSignMessage>
```